.circleci/config.yml

```diff
@@ -69,6 +69,9 @@ jobs:
             export PATH=$PWD/firefox:$PATH
             make test
 
+      - store_artifacts:
+          path: /home/circleci/repo/build/
```

Comment:
Cool.  I didn't realize this would be so easy.  This reminds me I need to get auto-deployment working again.